src/app/members/components/MemberFormBuilder.tsx

```diff
@@ -132,14 +139,16 @@ export const MemberFormBuilder = () => {
 
   const handleSubmission = async (values: UserBuilder) => {
     try {
-      // there is nothing in response in case of insert data
-      const result = await createUser(values);
+      const result = member
+        ? await updateUser(values, member.id)
+        : await createUser(values);
 
       if (!result) {
-        form.reset();
         toast({
-          title: "User created successfully",
-          description: "You will receive message shortly",
+          title: member?.id
+            ? "User updated successfully"
+            : "User created successfully",
+          description: member?.id ? "" : "You will receive message shortly",
```

Comment:
Description doesn't make sense. Leave it empty if not needed.

Reply:
fixed